src/test/kotlin/no/nav/fia/arbeidsgiver/samarbeidsstatus/api/SamarbeidsstatusTest.kt

```diff
@@ -7,19 +7,25 @@ import io.ktor.http.HttpHeaders
 import io.ktor.http.HttpStatusCode
 import kotlinx.coroutines.runBlocking
 import kotlinx.serialization.json.Json
-import no.nav.fia.arbeidsgiver.helper.AltinnProxyContainer.Companion.ALTINN_ORGNR_1
-import no.nav.fia.arbeidsgiver.helper.AltinnProxyContainer.Companion.ALTINN_ORGNR_2
-import no.nav.fia.arbeidsgiver.helper.AltinnProxyContainer.Companion.ORGNR_UTEN_TILKNYTNING
+import no.nav.fia.arbeidsgiver.helper.AltinnTilgangerContainerHelper.Companion.ALTINN_ORGNR_1
+import no.nav.fia.arbeidsgiver.helper.AltinnTilgangerContainerHelper.Companion.ALTINN_ORGNR_2
+import no.nav.fia.arbeidsgiver.helper.AltinnTilgangerContainerHelper.Companion.ORGNR_UTEN_TILKNYTNING
 import no.nav.fia.arbeidsgiver.helper.TestContainerHelper
+import no.nav.fia.arbeidsgiver.helper.TestContainerHelper.Companion.altinnTilgangerContainerHelper
 import no.nav.fia.arbeidsgiver.helper.TestContainerHelper.Companion.applikasjon
 import no.nav.fia.arbeidsgiver.helper.TestContainerHelper.Companion.kafka
 import no.nav.fia.arbeidsgiver.helper.performGet
 import no.nav.fia.arbeidsgiver.helper.withTokenXToken
+import no.nav.fia.arbeidsgiver.samarbeidsstatus.api.AltinnTilgangerService.Companion.ENKELRETTIGHET_FOREBYGGE_FRAVÆR_I_ALTINN
 import no.nav.fia.arbeidsgiver.samarbeidsstatus.api.dto.SamarbeidsstatusDTO
 import no.nav.fia.arbeidsgiver.samarbeidsstatus.api.dto.Samarbeidsstaus
+import org.junit.Before
 import kotlin.test.Test
 
 class SamarbeidsstatusTest {
+    @Before
+    fun cleanUp() = runBlocking { altinnTilgangerContainerHelper.slettAlleRettigheter() }
+
```

Comment:
Savner en test som dekker følgende use-case: 
 - innlogget bruker som ikke har enkeltrettighet for å hente samarbeidsstatus (men har tilgang til virksomheten via en annen vilkårlig enkeltrettighet)
 - brukeren gjør oppslag mot endepunktet
 - applikasjon svarer 403